cmd/config/public.go

```diff
@@ -185,5 +187,14 @@ func GetMachineMetadata() map[string]string {
 }
 
 func GetMachineUserdata() string {
-	return viper.GetString(config.KeyInstanceUserdata)
+	s := viper.GetString(config.KeyInstanceUserdata)
+	return tryDecodeBase64(s)
+}
+
+func tryDecodeBase64(s string) string {
```

Comment:
I'd prefer this function name not to start with the word `try`. What was wrong with just `decodeBase64`?